packages/manager/modules/pci/src/projects/project/kubernetes/components/update-policy-picker/index.js

```diff
@@ -0,0 +1,14 @@
+import angular from 'angular';
+import angularTranslate from 'angular-translate';
+import uiKit from '@ovh-ux/ui-kit';
+
+import component from './update-policy-picker.component';
+
+const moduleName = 'ovhManagerPciKubeUpdatePolicyPicker';
+
+angular
+  .module(moduleName, [angularTranslate, uiKit])
+  .component('kubeUpdatePolicyPicker', component)
```

Comment:
```suggestion
  .component('pciProjectKubeUpdatePolicyPicker', component)
```

Reply:
Mmm I disagree @anooparveti 
The plan is to make kubernetes a µapp so let's start right now by having the correct name for new components right ?

Reply:
Nope, i dont agree. As long as we are in monolith, we have to follow it's conventions.